packages/nx-plugin/package.json

```
@@ -0,0 +1,69 @@
+{
+  "name": "@swarmion/nx-plugin",
+  "description": "Generate and use type-safe contracts between your Serverless services.",
+  "version": "0.4.4",
+  "contributors": [
+    "MaximeVivier",
+    "adriencaccia",
+    "guillaumeduboc"
+  ],
+  "license": "MIT",
+  "homepage": "https://github.com/swarmion/swarmion#readme",
+  "bugs": "https://github.com/swarmion/swarmion/issues",
+  "repository": "swarmion/swarmion.git",
+  "keywords": [
+    "Serverless",
+    "Typescript",
+    "generator",
+    "nx",
+    "nxplugin"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "sideEffects": false,
+  "files": [
+    "dist",
+    "generators.json"
+  ],
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/types/index.d.ts",
+  "generators": "./generators.json",
+  "scripts": {
+    "lint-fix": "yarn linter-base-config --fix",
+    "lint-fix-all": "yarn lint-fix .",
+    "linter-base-config": "eslint --ext=js,ts .",
+    "package": "rm -rf dist && yarn package-cjs && yarn package-esm && yarn package-types",
+    "package-cjs": "NODE_ENV=cjs yarn transpile --out-dir dist/cjs",
+    "package-esm": "NODE_ENV=esm yarn transpile --out-dir dist/esm",
+    "package-types": "ttsc -p tsconfig.build.json",
+    "test": "yarn test-linter && yarn test-type",
+    "test-linter": "yarn linter-base-config .",
+    "test-type": "tsc --noEmit --emitDeclarationOnly false",
+    "transpile": "babel src --extensions .ts --quiet --source-maps --copy-files",
+    "watch": "rm -rf dist && concurrently 'yarn:package-* --watch'"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.17.6",
+    "@babel/core": "^7.17.8",
+    "@babel/plugin-transform-runtime": "^7.17.0",
+    "@babel/preset-env": "^7.16.11",
+    "@babel/preset-typescript": "^7.16.7",
+    "@nrwl/devkit": "^13.9.5",
+    "@nrwl/js": "^13.9.5",
+    "@nrwl/linter": "^13.9.5",
+    "@swarmion/configuration": "workspace:*",
```

This will not work with lerna for publishing
```suggestion
    "@swarmion/configuration": "^0.4.5",
```

---

we still use lerna for publishing and not yarn?